sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -311,7 +311,9 @@ export async function createFunctionInfoAsync(
  */
 function createFunctionInfo(
         func: Function, context: Context,
-        serialize: (o: any) => boolean): FunctionInfo {
+        serialize: (o: any) => boolean, logInfo?: boolean): FunctionInfo {
+
+    // logInfo = logInfo || func.name === "addHandler";
```

Comment:
Useful for being able to turn on logging when serializing just a specific function.

Reply:
Should probably leave this out or formalize it as part of the debug logging for this package.